pkglistgen.py

```diff
@@ -69,7 +70,7 @@ def __init__(self, name, pkglist):
         self.solved = False
         self.not_found = dict()
         self.unresolvable = dict()
-        for a in self.architectures:
+        for a in ARCHITECTURES:
```

Comment:
should be the specified architecures here

Reply:
no, thats the fix.. in my case specified architectures is aarch64, but the package group contained x86_64, which resulted in keyerror.